### PR TITLE
don't skip the migration of bot accounts

### PR DESCRIPTION
**Pull request by markobrien1**
Signed-off-by: Mark O'Brien <markobri@redhat.com>

### CODE REVIEW

**Comment by markobrien1** [comment]
I am waiting for confirmation that we would like all bot accounts before this being merged

File: fas2ipa/users.py [hunk 1]
```diff
@@ -233,7 +233,11 @@ def _compact_value(cls, val):
     def migrate_user(self, person):
         person_orig = person
         person = person.copy()
-        if self.config["users"]["skip_disabled"] and person.get("status") != "active":
+        if (
+                self.config["users"]["skip_disabled"] 
+                and person.get("status") != "active" 
+                and person.get("status") != "bot"
```
Comment (nphilipp) [inline fas2ipa/users.py]: To make this more concise: `... and person.get("status") not in ("active", "bot") ...`

**Review by nphilipp — COMMENTED** [review]
I think that if we run this, we should filter the accounts to only migrate the few bot accounts we have rather than letting this loose on everybody's account and potentially overwriting already changed values with whatever is in FAS.

File: fas2ipa/users.py (hunk 1 above)
Comment (markobrien1) [reply fas2ipa/users.py]: Ok the latest commit should fix that

**Pull request merged**